v20.2/performance-benchmarking-with-tpc-c-100k-warehouses.md

```diff
@@ -11,16 +11,22 @@ This page shows you how to reproduce [CockroachDB's TPC-C performance benchmarki
 <div class="filters filters-big clearfix">
   <a href="performance-benchmarking-with-tpc-c-10-warehouses.html"><button class="filter-button">10</button></a>
   <a href="performance-benchmarking-with-tpc-c-1k-warehouses.html"><button class="filter-button">1000</button></a>
-  <a href="performance-benchmarking-with-tpc-c-10k-warehouses.html"><button class="filter-button">10,000</button></a>
-  <button class="filter-button current"><strong>100,000</strong></button>
+  <a href="performance-benchmarking-with-tpc-c-10k-warehouses.html"><button class="filter-button">13,000</button></a>
+  <button class="filter-button current"><strong>140,000</strong></button>
 </div>
 
-Warehouses | Data size | Cluster size
------------|-----------|-------------
-10 | 2GB | 3 nodes on your laptop
-1000 | 80GB | 3 nodes on `c5d.4xlarge` machines
-10,000 | 800GB | 15 nodes on `c5d.4xlarge` machines
-100,000 | 8TB | 81 nodes on `c5d.9xlarge` machines
+<strong><font style="size:xx-large" color="red">FIXME</font></strong>: Update data size column in table below
+
+| Cluster size                       | Data size | Warehouses |
+|------------------------------------+-----------+------------|
+| 3 nodes on your laptop             | 2GB       | 10         |
+| 3 nodes on `c5d.4xlarge` machines  | 80GB      | 1000       |
+| 15 nodes on `c5d.4xlarge` machines | 800GB     | 13,000     |
+| 81 nodes on `c5d.9xlarge` machines | 8TB       | 140,000    |
+
+<span class="version-tag">New in v20.2</span>: CockroachDB can achieve a TPC-C run of 140k warehouses on the same cluster size used for 100k in version 19.2, a 42% improvement:
+
+<img src="{{ 'images/v20.2/tpcc140k.png' | relative_url }}" alt="TPC-C 140,000" style="max-width:100%" />
```

Comment:
On this page and the 13k page, I added a note and the chart image from Vy's spreadsheet comparing the results between 19.2 and 20.2 that describes the improvements (30% and 42% respectively).